web/app/view/ReportController.js

```diff
@@ -77,6 +90,152 @@ Ext.define('Traccar.view.ReportController', {
     },
 
     selectReport: function (position, center) {
-        this.getView().getSelectionModel().select([position], false, true);
+        if (position instanceof Traccar.model.Position) {
+            this.getView().getSelectionModel().select([position], false, true);
+        }
+    },
+
+    downloadCsv: function (requestUrl, requestParams) {
+        Ext.Ajax.request({
+            url: requestUrl,
+            method: 'GET',
+            params: requestParams,
+            headers: {
+                Accept: 'text/csv'
+            },
+            success: function (response) {
```

Comment:
Unfortunately, there is no regular way. Every way is a trick. (hidden iframes, self clicked links and so on)
Simpliest way is hiden `Ext.form.Panel`. Everything does ExtJS for us, but it replaces header to his own.

We can change server side to response with csv by default.